source/features/sync-pr-commit-title.tsx

```diff
@@ -6,6 +6,7 @@ import * as textFieldEdit from 'text-field-edit';
 
 import features from '.';
 import onPrMergePanelOpen from '../github-events/on-pr-merge-panel-open';
+import concatRegex from '../helpers/concat-regex';
```

Comment:
We try to keep the `from` keywords "aligned" so it's easier to follow them
```suggestion
import concatRegex from '../helpers/concat-regex';
import onPrMergePanelOpen from '../github-events/on-pr-merge-panel-open';
```